Vejleder rettelser (#14)
chhoumann/p5: sections/Chapter 5 Sprint 3/Sprint3.tex

\chapter{Sprint 3}

> This chapter covers the development process during the 3rd sprint. The 3rd sprint covers the the implementation of the needed database features for getting a functioning search engine. 

Reviewer: ```suggestion
This chapter covers the development process during the third sprint. The third sprint covers the implementation of the needed database features for getting a functioning search engine. 
```

> The overall goal for the Knox project during this sprint was to get the search engine to work. To accomplish this goal the product owner specified a list of expectations for the entire \knox project, that when completed would fufill a minimal viable product. The specified list of common goals is outlined below.

Reviewer: ```suggestion
To accomplish this goal, the product owner specified a list of expectations for the entire \knox{} project that, when completed, would fufill a minimal viable product. The specified list of common goals is outlined below.
```

Reviewer: No need to repeat that this is the goal.

\begin{itemize}
	\item It should be possible to make searches in all the available databases.
	\item It should be possible to make searches using keywords in all the available data.
	\item It should be possible to make searches in all Nordjyske articles from 2017-2021
	\item It should be possible to make searches in all the Grundfos manuals, barring those that have shown to be problematic during processing.

> 	\item The UI should minimally be able to display a link to the given article but preferably display the title and text.

Reviewer: ```suggestion
	\item The UI should at least be able to display a link to the given article but preferably also display the title and text.
```
